### PR TITLE
sbt-echo and inspect-related fixes

### PR DESCRIPTION
**Pull request by corruptmemory**
* Rebase off of wip/sbt-server
* Updated versions of upstream artifacts
  Play: 2.3.8-M1
  Akka: 2.3.9

Tracing works but activator is currently not operating correctly (can't actually run a Play project from the UI, but from console I have evidence that tracing is working).

```
background log: info: --- (Running the application, auto-reloading is enabled) ---
background log: info: 
background log: info: play - Listening for HTTP on /0:0:0:0:0:0:0:0:9000
background log: info: 
background log: info: (Server started, use Ctrl+D to stop and go back to the console...)
background log: info: 
background log: info: Starting ProtobufTraceSender:
background log: info: Starting ProtobufTraceSender:
background log: info: LESS compiling on 1 source(s)
background log: info: CoffeeScript compiling on 1 source(s)
background log: info: play - Application started (Dev)
background log: info: ProtobufTraceSender: trying to connect - localhost/127.0.0.1:28667
background log: info: ProtobufTraceSender: FAILED to connect to collector - localhost/127.0.0.1:28667
background log: info: WARN: No trace receiver on port [28667] 
background log: info: ProtobufTraceSender: trying to connect - localhost/127.0.0.1:28667
background log: info: ProtobufTraceSender: FAILED to connect to collector - localhost/127.0.0.1:28667
background log: info: WARN: No trace receiver on port [28667] 
background log: info: play - Starting application default Akka system.
```

I've not published `echo` or `sbt-echo` so you need to publish a version locally (the code is looking for version `0.1.8`)

Inspect needs to look something like this:

```scala
addSbtPlugin("com.typesafe.play" % "sbt-fork-run-plugin" % "2.3.8-M1")

addSbtPlugin("com.typesafe.sbt" % "sbt-echo-play" % "0.1.8")
```

As a warning, right now git SHA versions of things are evicting `ui-interface-0-13:1.0-M1` for `ui-interface-0-13:1.0-b3f1fbe7496a2b58b856460e3a8b50d0db988c63`

### CODE REVIEW

File: echo/trace/core/src/main/scala/com/typesafe/trace/Trace.scala [hunk 1]
```diff
@@ -72,7 +76,39 @@ object Trace {
...
+      "remoteLifeCycle: " + remoteLifeCycle,
+      "useDispatcherMonitor: " + useDispatcherMonitor,
+      "dispatcherPollInterval: " + dispatcherPollInterval,
+      "useSystemMetricsMonitor: " + useSystemMetricsMonitor,
+      "systemMetricsPollInterval: " + systemMetricsPollInterval,
+      "deadlockWatchFrequency: " + deadlockWatchFrequency,
+      "shutdownTimeout: " + shutdownTimeout).mkString("\n")
+
+    def maybeDumpConfig(): Unit =
+      if (reportConfig) println("Trace settings configuartion:\n" + reportString)
```
Comment (henrikengstrom) [inline echo/trace/core/src/main/scala/com/typesafe/trace/Trace.scala]: s/configuartion/configuration

File: echo/trace/core/src/main/scala/com/typesafe/trace/send/ProtobufTraceSender.scala [hunk 2]
```diff
@@ -62,19 +64,23 @@ class ProtobufTraceSender(port: Int, capacity: Int, retry: Boolean, daemonic: Bo
   }
 
   private def connect(): Boolean = {
+    println("ProtobufTraceSender: trying to connect - " + address + ":" + port)
```
Comment (henrikengstrom) [inline echo/trace/core/src/main/scala/com/typesafe/trace/send/ProtobufTraceSender.scala]: Do we always want to print this (and the lines below)?

Comment (corruptmemory) [inline echo/trace/core/src/main/scala/com/typesafe/trace/send/ProtobufTraceSender.scala]: Not sure.  It doesn't produce all that much noise on output.  AppDynamics and NewRelic's agents print out something to indicate that they are working.

Comment (henrikengstrom) [inline echo/trace/core/src/main/scala/com/typesafe/trace/send/ProtobufTraceSender.scala]: No biggie. We can always remove it if we think things get too noisy.

**Pull request merged**